scripts/eval/rag_evaluator/marlin_accuracy.py

```
@@ -0,0 +1,148 @@
+                    temperature=0.1
+                ).content
+                score2 = self.process_score(score2)
+                if score2 == score2:
+                    break
+
+            # Average scores or discard failed score
+            score = self.average_scores(score1, score2)
+
+            return score
+
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)